src/Microsoft.Azure.Functions.Extensions.Mcp/Abstractions/IMcpToolProperty.cs

```diff
@@ -12,4 +12,6 @@ public interface IMcpToolProperty
     public string? Description { get; set; }
     
     public bool Required { get; set; }
+
+    public bool IsArray { get; set; }
```

Comment:
Still want to keep this array. Debated renaming `Required` to `IsRequired`, but there'd be impact to workers. We can still deal with the breaking change if needed (likely in a separate PR)